dockstore-webservice/src/main/resources/queries/mapping_notebook.json

```diff
@@ -18,6 +18,13 @@
             "stopwords": [ "https", "http", "see", "from", "use", "usage", "more", "can", "reads", "website", "count" ]
           }
         },
+        "char_filter": {
+          "underscores_to_spaces": {
+            "type": "pattern_replace",
+            "pattern": "_",
```

Comment:
Not necessarily an objection, but what does a user do when they legitimately want to search for something with underscores? Will/can putting something in quotes avoid this filter?

Reply:
During testing, I was concerned about this, so I ran some searches on the underscored term ("samtools_index"), and ES seemed to do the right thing (return hits that contained "samtools_index").  I'm not familiar with the ES internals, so unclear exactly what's going on, but the takeaway is that the behavior seems to be what we want...